helm/hpcc/templates/network.yaml

```diff
@@ -157,6 +157,25 @@ spec:
         matchLabels:
           app: eclservices
 ---
+{{ range .Values.esp -}}
+{{- if has .name $.Values.global.visibilities.exposedServices }}
```

Comment:
minor: This will cause an error if "visibilities" isn't defined.
Elsewhere we check for it's existence before accessing keys in it, e.g. in "hpcc.isVisibilityPublic".

Reply:
Seems wrong to be specifying exposed services via global.visibilities. I would have thought the settings in the relevant visibility section should be sufficient

Reply:
So each service define its own ingress as:

> esp:
> \- name: eclwatch
>     service:
>         ingress:
>         \- {}